spec/components/bali/field_group_wrapper_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+class TestHelper < ActionView::Base; end
```

Comment:
Do you still need this?
I think you are using the TestHelper from `spec/dummy/app/helpers/test_helper.rb`